### PR TITLE
Make improvements to building a data service guide

### PR DESCRIPTION
**Pull request by kaneeldias**
## Purpose
$subject

Fixes: https://github.com/ballerina-platform/ballerina-dev-website/issues/4880

### CODE REVIEW

File: swan-lake/featured-use-cases/build-a-data-service-in-ballerina.md [hunk 1]
```diff
@@ -179,6 +186,9 @@ The `mysql:Client` provides two primary remote methods for performing queries.
 Use `query()`, `queryRow()` and `execute()` methods to define methods that can perform basic CRUD operations against the MySQL database.
 
 ```ballerina
+import ballerina/sql;
```
Comment (praneesha) [inline swan-lake/featured-use-cases/build-a-data-service-in-ballerina.md:189]:
```suggestion
import ballerina/sql;
```
Don't we need to remove this unused import?

File: swan-lake/featured-use-cases/build-a-data-service-in-ballerina.md [hunk 2]
```diff
@@ -259,7 +269,7 @@ import ballerina/http;
 
 ### Create a service
 Now, you can create a service as shown below. This creates an `/employees` endpoint on port `8080` which can 
-be accessed via a browser by visiting `http://locahost:8080/employees` after executing the command `bal run`.
+be accessed by visiting `http://localhost:8080/employees` after executing the command `bal run`.
```
Comment (praneesha) [inline swan-lake/featured-use-cases/build-a-data-service-in-ballerina.md:272]:
```suggestion
be accessed by visiting `http://localhost:8080/employees` after executing the `bal run`  command.
```

File: swan-lake/featured-use-cases/build-a-data-service-in-ballerina.md [hunk 3]
```diff
@@ -284,9 +294,11 @@ service /employees on new http:Listener(8080) {
 Invoke the defined resource function by sending a `POST` request to `http://localhost:8080/employees` with the 
 required data as a JSON payload.
 
+
 ```shell
+# Using a Unix terminal
```
Comment (praneesha) [inline swan-lake/featured-use-cases/build-a-data-service-in-ballerina.md:299]:
```suggestion
# Using a Unix terminal
```
Is this a level 1 heading?

File: swan-lake/featured-use-cases/build-a-data-service-in-ballerina.md [hunk 4]
```diff
@@ -301,6 +313,27 @@ curl --location --request POST 'http://localhost:8080/employees/' \
         "manager_id": 1,
         "job_title": "Sales Manager"
     }'
+    
+    
+# Using a Windows Powershell terminal
```
Comment (praneesha) [inline swan-lake/featured-use-cases/build-a-data-service-in-ballerina.md:318]:
```suggestion
# Using a Windows Powershell terminal
```
Is this a level 1 heading?

File: swan-lake/featured-use-cases/build-a-data-service-in-ballerina.md (hunk 1 above)
Comment (kaneeldias) [reply swan-lake/featured-use-cases/build-a-data-service-in-ballerina.md:189]: The unused import was in a previous sample. It has been removed from there.
https://github.com/ballerina-platform/ballerina-dev-website/pull/5041/files#diff-13e9db3064102e2d1ec06a2f2b92cbdb93f195684ceb41c21418864b92813937L152

File: swan-lake/featured-use-cases/build-a-data-service-in-ballerina.md (hunk 3 above)
Comment (kaneeldias) [reply swan-lake/featured-use-cases/build-a-data-service-in-ballerina.md:299]: It's a comment inside a codeblock

File: swan-lake/featured-use-cases/build-a-data-service-in-ballerina.md (hunk 4 above)
Comment (kaneeldias) [reply swan-lake/featured-use-cases/build-a-data-service-in-ballerina.md:318]: It's a comment inside a codeblock

File: swan-lake/featured-use-cases/build-a-data-service-in-ballerina.md (hunk 2 above)
Comment (kaneeldias) [reply swan-lake/featured-use-cases/build-a-data-service-in-ballerina.md]: Fixed in [82417aa](https://github.com/ballerina-platform/ballerina-dev-website/pull/5041/commits/82417aa7d92ccb948ab137f2b8ee79856a286b0c)

**Review by praneesha — APPROVED** [review]

**Pull request merged**